src/Calculator.ts

```diff
@@ -273,15 +274,16 @@ export function calculateMeasureReports(
       report.group?.push(group);
     });
 
-    // TODO: create contained evaluatuated resource (contains patient... and population and entity information?)
-    // may eventually need the other bits (all pieces of information that were used in calculation), but we might be able to ignore for now
-    const evalId = uuidv4();
-    const contained: R4.IBundle = {
-      resourceType: 'Bundle',
-      id: evalId,
-      type: R4.BundleTypeKind._collection,
-      entry: []
-    };
+    if (result.evaluatedResources) {
```

Comment:
Is there somewhere that these evaluated resources gets pushed to the report's `contained` resources so that the references can be looked up from the report?

Reply:
So I considered two approaches.

A) Push them onto contained and reference them with `#id` in evaluated
B) Leave them off of contained and reference them with `ResourceType/id` in evaluated

the latest version of cqf-ruler uses B), so that's what I went with. It's easy to change. It's something I was considering bringing up at the connectathon as well, but I'm partial to the resource type/id format for now.